python/aida_interchange/aifutils.py

```diff
@@ -332,9 +332,10 @@ def make_cluster_with_prototype(g, cluster_uri, prototype, system):
     """
     cluster = _make_aif_resource(g, cluster_uri, AIDA_ANNOTATION.SameAsCluster, system)
     g.add((cluster, AIDA_ANNOTATION.prototype, prototype))
+    if handle == None:
```

Comment:
I know I'm not put as a reviewer, but shouldn't this be !=?

Reply:
Yes it should.  Typo.  Thanks.